sdk/storage/azure-storage-queue/samples/queue_samples_authentication_async.py

```diff
@@ -110,8 +110,8 @@ async def authentication_by_shared_access_signature_async(self):
         from azure.storage.queue import generate_account_sas, ResourceTypes, AccountSasPermissions
 
         sas_token = generate_account_sas(
-            queue_service.account_name,
-            queue_service.credential.account_key,
+            self.account_name,
+            self.access_key,
```

Comment:
This didn't require a `None` check because `queue_samples_authentication.py` used `self.account_name` and `self.access_key` directly, and the `None` check takes place above already.